lib/src/core/better_player_with_controls.dart

```diff
@@ -158,18 +158,40 @@ class _BetterPlayerWithControlsState extends State<BetterPlayerWithControls> {
     BuildContext context,
     BetterPlayerController betterPlayerController,
   ) {
-    if (controlsConfiguration.showControls &&
-        controlsConfiguration.customControls == null) {
-      if (Platform.isAndroid) {
-        return BetterPlayerMaterialControls(
-          onControlsVisibilityChanged: onControlsVisibilityChanged,
-          controlsConfiguration: controlsConfiguration,
-        );
+    if (controlsConfiguration.showControls) {
+      if (controlsConfiguration.customControls == null &&
+          controlsConfiguration.playerPlatform == null) {
+        if (Platform.isAndroid) {
+          return BetterPlayerMaterialControls(
```

Comment:
Please create 2 methods which will create Material and Cupertino controls when requested. I believe we have code duplication here.

Reply:
Also based on comment above, when we set PlayerTheme default value for given platform, we don't need to react here. So we dont  this if block:
``` if (controlsConfiguration.showControls) {
      if (controlsConfiguration.customControls == null &&
          controlsConfiguration.playerPlatform == null) {
        if (Platform.isAndroid) {
...
```

So we need only this if block:
```
if (showControls){
	if (customControls){
		return buildCustomControls();
	} else if (platformTheme == material){
		return buildMaterialControls();
	} else if (platformTheme == cupertino){
		return buildCupertinoControls();
	}
} 
```